### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by consideRatio**
I'm not sure how to link the "Learn about our infrastructure text" link that was broken, but guessed https://docs.2i2c.org/en/latest/ - please chime in on that!

### CODE REVIEW

File: config/_default/menus.toml [hunk 1]
```diff
@@ -65,5 +65,5 @@
 
 [[main_right]]
   name = 'support'
-  url = "https://docs.2i2c.org/en/latest/admin/howto/support.html"
+  url = "https://2i2c.freshdesk.com/"
```
Comment (damianavila) [inline config/_default/menus.toml]: Maybe this one instead? https://docs.2i2c.org/en/latest/support.html
@choldgraf do you have any thoughts on this one?

File: content/service/jupyter-cta.md [hunk 2]
```diff
@@ -22,6 +22,6 @@ While Jupyter is at the core of 2i2c's infrastructure, Jupyter is flexible enoug
 
 <div class="cta-group">
 
-{{< cta cta_text="Learn about our infrastructure" cta_link="https://docs.2i2c.org/en/latest/admin/index.html" cta_new_tab="true" >}}
+{{< cta cta_text="Learn about our infrastructure" cta_link="https://docs.2i2c.org/en/latest/" cta_new_tab="true" >}}
```
Comment (damianavila) [inline content/service/jupyter-cta.md:25]: Makes sense to me, @consideRatio

File: config/_default/menus.toml (hunk 1 above)
Comment (choldgraf) [reply config/_default/menus.toml]: ahh that's a good point @damianavila - that page has more context, agreed we should use that one instead

Comment (choldgraf) [inline config/_default/menus.toml]:
```suggestion
  url = "https://docs.2i2c.org/en/latest/support.html"
```

**Comment by choldgraf** [comment]
this looks great to me, thanks so much @consideRatio :-)

**Pull request merged**